<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.CFcHL19' shared service.

return $this->privates['.service_locator.CFcHL19'] = new \Symfony\Component\DependencyInjection\ServiceLocator(array('agruparproductoRepository' => function (): \App\Repository\AgruparproductoRepository {
    return ($this->privates['App\Repository\AgruparproductoRepository'] ?? $this->load('getAgruparproductoRepositoryService.php'));
}, 'ingreso' => function (): \App\Entity\Ingresos {
    return ($this->privates['.errored..service_locator.CFcHL19.App\Entity\Ingresos'] ?? $this->load('getIngresos2Service.php'));
}));
